Public/Src/Engine/UnitTests/Processes.Detours/SandboxedProcessPipExecutorTest.cs

```diff
@@ -1876,6 +1876,88 @@ public async Task ProcessUsingUntrackedEnvironmentVariable()
             }
         }
 
+        [Theory]
+        [InlineData(false)]
+        [InlineData(true)]
+        public async Task ProcessGlobalUntrackedScope(bool translate)

```

Comment:
>ProcessGlobalUntrackedScope [](start = 26, length = 27)

Writing Detours test only exercises Windows case. Why don't you write an integration test, so that both Windows and Mac are covered?

Reply:
moved to integration test.